CriticalMass/BikeAnnotationView.swift

```diff
@@ -26,16 +34,12 @@ class BikeAnnoationView: MKAnnotationView {
     }
 }
 
-class FriendAnnotationView: MKAnnotationView {
+class FriendAnnotationView: MKAnnotationView, AnnotationView {
+    typealias T = Friend
```

Comment:
I think it should not be named `T` since this is not a Generic and it mixes things up